operators/samples/tenant.yml

```
@@ -1,7 +1,7 @@
 apiVersion: crownlabs.polito.it/v1alpha1
 kind: Tenant
 metadata:
-  name: mariorossi
+  name: mario.rossi
```

```suggestion
  name: john.doe
```
(better to use English terms)